sdk/search/search/review/search.api.md

```diff
@@ -294,18 +294,28 @@ export interface Index {
 }
 
 // @public
-export type IndexAction<T> = {
-    actionType: IndexActionType;
-} & Partial<T>;
+export type IndexActionType = 'upload' | 'merge' | 'mergeOrUpload' | 'delete';
 
 // @public
-export type IndexActionType = 'upload' | 'merge' | 'mergeOrUpload' | 'delete';
+export type IndexDocumentAction<T> = {
```

Comment:
Do we want `Document` or `Documents` here?  Everything else is `Documents`, but this obviously applies to just one.  We have `IndexDocumentsAction` in .NET for consistency with all the other APIs (and so they align in intellisense).

Reply:
I'm fine with the plural

Reply:
but since the action is on a single document...

Reply:
@bterlson naming bikeshed for you

Reply:
One could argue that `Action` is singular and it's the action corresponding to an `IndexDocuments` operation...  but I don't feel very strongly about this.

Reply:
IndexDocumentsAction is fine by me, but I also don't feel very strongly.

Reply:
Let's align here.